Scripts/Python/xAgeSDLBoolActivatorComboSet.py

```diff
@@ -194,9 +210,20 @@ def _RespPushComplete(self, events):
         self._HitActId = None
 
     def _TriggerButton(self, actId):
+        self._AddToAttempt(actId)
+
         if self._solved:
             PtDebugPrint("xAgeSDLBoolActComboSet._TriggerButton():\tYou just unsolved it, moron.", level=kWarningLevel)
             self._solved = False
+            self._numCorrect = 0
```

Comment:
```suggestion
```

This is done implicitly in `OnSDLNotify()`.

Reply:
I added it because it was also being done down on line 242: https://github.com/H-uru/Plasma/pull/1194/files#diff-675dbaccdec1487a34621a04230d97bd6bae77cd670d6f99d50593ad5c29713aR242

It seemed like it should be consistent, unless there's something different about the context that necessitates one being set explicitly. I'll remove it here, but should I also remove it there as well?

Reply:
On line 242, the SDL solved is probably already false, so the `OnSDLNotify()` won't be fired (no-op changes don't fire notifications), so the explicit set is needed.